server/multitenancy/tenant_resolver.ts

```diff
@@ -80,6 +80,8 @@ export function isMultitenantPath(request: KibanaRequest): boolean {
     request.url.pathname?.startsWith('/elasticsearch') ||
     request.url.pathname?.startsWith('/api') ||
     request.url.pathname?.startsWith('/app') ||
+    // bootstrap.js depends on tenant info to fetch kibana configs in tenant index
+    (request.url.pathname?.indexOf('bootstrap.js') || -1) > -1 ||
```

Comment:
`request.url.pathname?.includes('bootstrap.js')` should works. String.includes returns a boolean.

Reply:
ts compiler complains that `request.url.pathname?.indexOf('bootstrap.js')` might be `undefined`, thus not comparable with `-1`, so write it this way

Reply:
I mean `includes`, it return boolean instead of number.